cmd/get-hardware-details/main.go

```diff
@@ -55,36 +55,36 @@ func main() {
 
 	ironic, err := clients.IronicClient(endpoint, opts.AuthConfig, tlsConf)
 	if err != nil {
-		fmt.Printf("could not get ironic client: %s", err)
+		log.Printf("could not get ironic client: %s", err)
 		os.Exit(1)
 	}
 
 	introData := nodes.GetInventory(context.TODO(), ironic, opts.NodeID)
 	data, err := introData.Extract()
 	if err != nil {
-		fmt.Printf("could not get inspection data: %s", err)
+		log.Printf("could not get inspection data: %s", err)
 		os.Exit(1)
 	}
 
 	json, err := json.MarshalIndent(hardwaredetails.GetHardwareDetails(data, klog.NewKlogr()), "", "\t")
 	if err != nil {
-		fmt.Printf("could not convert inspection data: %s", err)
+		log.Printf("could not convert inspection data: %s", err)
 		os.Exit(1)
 	}
 
-	fmt.Println(string(json))
+	log.Println(string(json))
```

Comment:
Here we may have to use nolint. I think the default logger prints to stderr, which is fine for everything else, but this line here is the output of the program and should go to stdout.

Reply:
Good catch, I tried looking for these cases, but missed it.

Reply:
/lgtm cancel

Reply:
I think we got two lines mixed up here. This should have nolint and still use fmt, line 73 should have Fatalf.
```suggestion
        //nolint:forbidigo
	fmt.Println(string(json))
```